config/jobs/image-pushing/k8s-staging-csi-gen.sh

```diff
@@ -0,0 +1,71 @@
+#! /bin/bash -e
```

Comment:
In our infra-related scripts at [`k/k8s.io`](https://github.com/kubernetes/k8s.io) we are trying to be consistent with [Google's shell styleguide ](https://google.github.io/styleguide/shellguide.html) so some of my comments will be related to this styleguide, but I'm not gonna block the PR because of this and leave you the decission if you would like to consider the suggestions.

As setting flags directly in shebangs can be broken when calling te script like:
```bash
bash script_name.sh
```
I would suggest
```suggestion
#!/usr/bin/env bash

set -o errexit
```